src/D_configure_taskserver.sh

```diff
@@ -1,197 +1,198 @@
 #!/bin/sh
-mkdir_taskddata() {
-	TASKDDATA=$1
-	output=$(mkdir -p $TASKDDATA)
-	echo $output
+mkdir_taskd_data() {
+	TASKD_DATA=$1
+	output=$(mkdir -p "$TASKD_DATA")
+	echo "$output"
 	# TODO: test whether the folder did not already exist
 }
 
 
-mkdir_taskddata_pki() {
-	TASKDDATA=$1
-	output=$(mkdir -p $TASKDDATA/pki)
-	echo $output
+mkdir_taskd_data_pki() {
+	TASKD_DATA=$1
+	output=$(mkdir -p "$TASKD_DATA"/pki)
+	echo "$output"
 	# TODO: test whether the folder did not already exist
 }
 
 
-initialise_taskserver() {
+initialise_task_server() {
 	output=$(taskd init)
-	echo $output
+	echo "$output"
 }
 
 # TODO: remove duplicate
-make_pki_dir_in_taskddata() {
-	TASKDDATA=$1
-	output=$(mkdir -p $TASKDDATA/pki)
-	echo $output
+make_pki_dir_in_taskd_data() {
+	TASKD_DATA=$1
+	output=$(mkdir -p "$TASKD_DATA"/pki)
+	echo "$output"
 }
 
-copy_pki_content_to_taskddata_dir() {
-	TASKDDATA=$1
-	TASKSERVER_TAR_NAME=$2
-	output=$(cp -r $TASKSERVER_TAR_NAME/pki* $TASKDDATA)
-	echo $output
+copy_pki_content_to_taskd_data_dir() {
+	TASKD_DATA=$1
+	TASK_SERVER_TAR_NAME=$2
```

Comment:
`TASK_SERVER_TAR_NAME` is a local variable, I think because:
1.  that is only used in this function (and/or its child functions).
2. https://google.github.io/styleguide/shellguide.html#s7.6-use-local-variables says: `al variables are only seen inside a function and its children by using local when declaring them`.

So if you could change that capitalized variable name to non-capitalized and could prepend `local ` to it, that would be great!

If you could repeat that change to all other local variables (except TASKDDATA), that would be awesome!

My appologies for nitpicking, I am learning from this myself as well! So thank you for your work and effort!